testing/scripts/conftest.py

```
@@ -79,16 +78,11 @@ def _suite_init(page, request):
-    if ssh_host and container and wp_path:
-        subprocess.run(
-            ["ssh", ssh_host,
-             f"docker exec {container} wp option delete swh_restrict_to_assigned"
-             f" --path={wp_path} --allow-root 2>/dev/null"],
-            capture_output=True, timeout=15
-        )
+    # Reset option that may be dirtied by a prior failed technician workflow test
+    try:
+        t.wpcli("option delete swh_restrict_to_assigned")
+    except Exception:
+        pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)